pkg/controller/datavolume-controller.go

```diff
@@ -1965,6 +2170,25 @@ func getName(storageClass *storagev1.StorageClass) string {
 	return ""
 }
 
+func getPreferredCloneStrategyForStorageClass(c client.Client, storageClass *storagev1.StorageClass) (*cdiv1.CDICloneStrategy, error) {
```

Comment:
Same here make it part of the reconciler, then you don't have to pass the client to it.

Reply:
done